src/python/WMComponent/AgentStatusWatcher/DrainStatusPoller.py

```diff
@@ -174,8 +174,8 @@ def checkSpeedDrainThresholds(self):
         # get the current speed drain status
         speedDrainConfig = self.agentConfig.get("SpeedDrainConfig")
 
-        # get condor jobs
-        jobs = self.condorAPI.getCondorJobs("", [])
+        # get a summary of the condor jobs
+        jobs = self.drainAPI.checkCondorStates(totalOnly=True)
```

Comment:
If my previous comment is to be taken into account, this call here should simply be converted to a call of a separate method, instead to a call of the same method with additional flag.